content/en/_index.md

```diff
@@ -163,6 +180,42 @@ public static void listInfoTypes() throws IOException {
   }
 }
 {{< /tab >}}
+{{< tab header="Python" >}}
+def list_info_type(
+    language_code: Optional[str] = "en-US",
+    result_filter: Optional[str] = "supported_by=INSPECT",
+):
+    """List types of sensitive information within a category.
+
+    Args:
+        language_code (str): The BCP-47 language code to use, e.g. 'en-US'.
+        filter (str): An optional filter to only return info types supported
+            by certain parts of the API. Supported filters are 
+            "supported_by=INSPECT" and "supported_by=RISK_ANALYSIS". Defaults
+            to "supported_by=INSPECT".
+    """
```

Comment:
The docstring comment here feels a bit verbose - do we really need it if there should be comment already at the top of the file?

Reply:
IMO, I think it's easier to follow next to each request. It lets me process them one at a time.